src/client/pages/Offer/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { BundledQuote } from 'data/graphql'
+import { Address } from 'pages/OfferNew/types'
+
+const parseAddress = (address: Address) => {
+  const { street, apartment, floor } = address
+
+  return street
+    .concat(floor ? `, ${floor}.` : '')
+    .concat(apartment ? ` ${apartment}` : '')
+}
+
+export const getAddress = (quotes: BundledQuote[]) => {
```

Comment:
I would rename this file to something less generic if it only has one export:

```
src/client/pages/Offer/utils.ts => src/client/pages/Offer/getAddress.ts
```

Reply:
I see. The idea is to use that file to gradually move `OfferNew/utils` content to it. I believe that we could use the same function in [InsuranceSummaryDetails](https://github.com/HedvigInsurance/web-onboarding/blob/master/src/client/pages/OfferNew/Checkout/InsuranceSummaryDetails.tsx#L207) when the migration is done.

But I change for something more specific as well, no problem :)